src/webhook/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9-1739801907 AS builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23 AS builder
```

Comment:
Same remark as with podvm-payload.

Reply:
I don't think it's needed: the upstream version already uses hardcoded image names, not variables.